knexfile.ts

```diff
@@ -0,0 +1,32 @@
+import type { Knex } from "knex";
+import path from "path";
+import dotenv from "dotenv";
+dotenv.config();
+
+const config: { [key: string]: Knex.Config } = {
+  development: {
+    client: 'sqlite3',
+      connection: {
+          filename: path.resolve(__dirname, 'src', 'database', 'database.sqlite')
```

Comment:
onde esse __dirname é definido?

Reply:
https://www.digitalocean.com/community/tutorials/nodejs-how-to-use__dirname